posthog/api/test/test_feature_flag.py

```diff
@@ -812,6 +812,43 @@ def test_updating_a_feature_flag_with_same_team_and_key_of_a_deleted_one(self):
         instance.refresh_from_db()
         self.assertEqual(instance.key, "alpha-feature")
 
+    def test_my_flags_is_not_nplus1(self):
+
+        query_counts: List[int] = []
+        queries: List[List[Dict[str, str]]] = []
+
+        count, qs = self._get_flags_counting_queries()
+        query_counts.append(count)
+        queries.append(qs)
+
+        # add new flags
+        for i in range(5):
```

Comment:
Would 2 be enough to verify scaling behaviour?

Reply:
(at least that it's constant)

Reply:
IME the counts don't necessarily scale linearly. Having more than 2 lets you check for that.